src/translations/fr.json

```diff
@@ -39,5 +46,16 @@
     "replaceList": "Remplacer la liste",
     "alertBeforeReplaceWithScript": "Voulez-vous vraiment remplacer cette liste par un script Groovy ?{br}{br}Cette action est irréversible",
     "remplacer": "Remplacer",
-    "copy": "Copier"
+    "copy": "Copier",
+    "newFilter": "Nouveau filtre",
+    "FilterName": "Nom",
+    "addNewFilter": "Créer un filtre",
+    "substationName": "Nom Poste",
```

Comment:
If it is 'substationName, french translation should be 'Nom Site'
If it is 'Nom Poste', the english translation should be 'VoltageLevel Name' and the key renamed